LINUX_keep_current.md

```diff
@@ -0,0 +1,285 @@
+
+# Keeping your setup current
+
+This section contains the steps that you need to follow in order to make sure that your setup is up to date
+
+First and foremost, in order to work in good conditions, make sure that :
+- you are working with a high speed internet connection
+- your computer has enough memory (8GB), so that you can run your code efficiently
+- your computer has enough disk space (30GB), so that you can work on big datasets
+
+## git
+
+Verify that git works
+
+``` bash
+git --version
+```
+
+👉 You should expect an output similar to this one yielding the git version :
+
+``` bash
+git version 2.33.0
+```
+
+## GitHub
+
+Verify that you have access to the Le Wagon public GitHub repositories
+
+``` bash
+cd ~/code/<YOUR_GITHUB_NICKNAME>/
+git clone git@github.com:lewagon/data-certification-exam data-certification-exam-test-delete-me
+```
+
+👉 The repo should clone correctly :
+
+``` bash
+Cloning into 'data-certification-exam-test-delete-me'...
+remote: Enumerating objects: 21, done.
+remote: Counting objects: 100% (21/21), done.
+remote: Compressing objects: 100% (14/14), done.
+Receiving objects: 100% (21/21), done.
+Resolving deltas: 100% (6/6), done.
+remote: Total 21 (delta 6), reused 16 (delta 1), pack-reused 0
+```
+
+👉 You can delete the cloned repo
+
+``` bash
+rm -Rf data-certification-exam-test-delete-me
+```
+
+## Verify your pyenv configuration
+
+Verify that you have a `~/.zprofile` :
+
+``` bash
+cat ~/.zprofile
+```
+
+👉 You should see the following lines :
+
+``` bash
+# Setup the PATH for pyenv binaries and shims
+export PYENV_ROOT="$HOME/.pyenv"
```

Comment:
According to the current `dotfiles`, these two exports are not anymore in `zshrc` but in `zprofile`.

If I don't have them on my `zprofile`, I get a super annoying message every time I open the a zsh shell.

Shouldn't we instead tell student to delete and move thses two `export` lines in the `zprofile`?